pydantic_ai_slim/pydantic_ai/models/anthropic.py

```diff
@@ -326,12 +345,21 @@ def _process_response(self, response: BetaMessage) -> ModelResponse:
                     )
                 )
 
+        # Map finish reason from stop_reason
+        mapped_finish = _map_anthropic_finish(getattr(response, 'stop_reason', None))
```

Comment:
Wy do we need `getattr` here? `response` is a `BetaMessage` which always has a `stop_reason`.